### DISCUSSION TITLE
How to change button style

### DISCUSSION THREAD

**Discussion by rlmma** [mmazzarolo/react-native-modal-datetime-picker#616] [Q&A] [answered]
Hello! I'm trying to find a way to change just the font weight of the "Cancel" button title. It looks bolder than the "Confirm" button title.

The only way I found so far is to use "customCancelButtonIOS" and create the whole new custom button element, but curious if there is any better and eaiser way to do it?

Thx!

**Comment by mmazzarolo** [comment, marked as answer]
Hey @Rimma-A ! 
Yeah, using `customCancelButtonIOS` **should** be the way to go. 
That said, we worked on making the UI of the component follow as close as possible the native iOS guidelines/components... so if you feel we can make changes to the font weight to get closer to the native iOS guidelines/components, we can discuss it and make the changes directly in the repo here (especially because we haven't updated the UI in ages, so we might need some UI updates) 👍 
Let me know!
